recipes/toolchain/deactivate.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
```

Comment:
> We should really try to clean stuff up so they don't have a surprise error down the line.

But that is what forks do for us. We aren't changing the environment of the process that actually called `conda build recipe`, just the environment within the conda-build subprocess call. No?